src/configuration.ts

```diff
@@ -11,7 +11,16 @@ const config: GlobalConfig = {
 	WEB3_POLLING_INTERVAL: 15000,
 	SUBGRAPH_POLLING_INTERVAL: 5000,
 	TOKEN_PRECISION: 2,
-
+	PRIMARY_NETWORK: {
+		name: 'Mainnet',
+		id: 1,
+		chain: '0x1',
+	},
+	SECONDARY_NETWORK: {
+		name: 'xDai',
+		id: 100,
+		chain: '0x64',
+	},
```

Comment:
we have these configurations as mainnet and xdai networks. let's use them.

Reply:
I'm a little confused on how to use this, I see you guys have two files, one for dev and the other for prod